README.md

```diff
@@ -6,6 +6,8 @@ programs**_ with tunable qubit interaction defined on _**arbitrary register topo
 [![pre-commit](https://github.com/pasqal-io/qadence/actions/workflows/lint.yml/badge.svg)](https://github.com/pasqal-io/qadence/actions/workflows/lint.yml)
 [![tests](https://github.com/pasqal-io/qadence/actions/workflows/test_fast.yml/badge.svg)](https://github.com/pasqal-io/qadence/actions/workflows/test_fast.yml)
 [![Build documentation](https://github.com/pasqal-io/qadence/actions/workflows/build_docs.yml/badge.svg)](https://pasqal-io.github.io/qadence/latest)
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Pypi]](https://badge.fury.io/py/qadence.svg)(https://pypi.org/project/qadence/)
 
 Documentation can be found [here](https://pasqal-io.github.io/qadence/latest).
```

Comment:
Please remove. The URL is already in the badge.

Reply:
?

Reply:
> ?

The URL to the docs is at least in three places, on the repo front page, in line 12 and in line 8. I think line 8 is sufficient.

Reply:
removed